api/testworkflows/v1/base_types.go

```diff
@@ -30,6 +30,9 @@ type TestWorkflowSpecBase struct {
 
 	// values to be used for test workflow execution
 	Execution *TestWorkflowTagSchema `json:"execution,omitempty" expr:"include"`
+
+	// Targets helps decide on which runner the execution is scheduled.
+	Targets *Target `json:"targets,omitempty" expr:"include"`
```

Comment:
Should it be `targets`, given it's a single target (+ in other APIs we have just `target`)?